### DISCUSSION TITLE
Launching multiple vehicles in JSBsim

### DISCUSSION THREAD

**Discussion by Judedarwinism** [JSBSim-Team/jsbsim#749] [Q&A] [answered]
can the JSBsim support multiple vehicles simulation? If so, is it that launch multiple instances of JSBsim like JSBSim::FGFDMExec FDMExec, JSBSim::FGFDMExec FDMExec1 and launch the corresponding aircraft?

**Comment by gallonmate** [comment, marked as answer]
Yes, see Sean's answer to the same question:
https://github.com/JSBSim-Team/jsbsim/discussions/426#discussioncomment-638596

**Reply by bomber64** [reply]
https://youtu.be/jnL5KglQG_g

**Comment by Judedarwinism** [comment]
I'm very appreciated for your answer!
